pkg/kubectl/plugins/plugins.go

```diff
@@ -46,6 +46,7 @@ type Plugin struct {
 // plugin as a command. Usually comes from a descriptor file.
 type Description struct {
 	Name      string  `json:"name"`
+	Use       string  `json:"use"`
```

Comment:
`Usage` is better?

Reply:
I agree, but `cobra.Command` calls it `Use`, so I didn't want to call it differently. But `Usage` is definitely clearer.